docs/components/typography/index.html

```diff
@@ -0,0 +1,77 @@
+<div class="typography-component">
+  <h1>Typography</h1>
+  <p class="text-lead margin-vertical-2">
+    Style your texts with different sets of typography.
+  </p>
+
+  <h2 class="margin-vertical-3">Display Sizes</h2>
+  <div class="typography-container margin-vertical-3">
+    <div>
+      <p class="font-size-xs">Extra Small Text</p>
+      <p class="font-size-sm">Small Text</p>
+      <p class="font-size-md">Medium Text</p>
+      <p class="font-size-lg">Large Text</p>
+      <p class="font-size-xl">Extra Large Text</p>
+      <p class="font-size-xxl">Extra Extra Large Text</p>
+    </div>
+  </div>
+  <iframe
+    src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C0%29&t=a11y-dark&wt=none&l=htmlmixed&width=919.25&ds=true&dsyoff=20px&dsblur=79px&wc=true&wa=false&pv=66px&ph=81px&ln=true&fl=1&fm=Hack&fs=16px&lh=136%25&si=false&es=2x&wm=false&code=%253Cp%2520class%253D%2522font-size-xs%2522%253EExtra%2520Small%2520Text%253C%252Fp%253E%250A%253Cp%2520class%253D%2522font-size-sm%2522%253ESmall%2520Text%253C%252Fp%253E%250A%253Cp%2520class%253D%2522font-size-md%2522%253EMedium%2520Text%253C%252Fp%253E%250A%253Cp%2520class%253D%2522font-size-lg%2522%253ELarge%2520Text%253C%252Fp%253E%250A%253Cp%2520class%253D%2522font-size-xl%2522%253EExtra%2520Large%2520Text%253C%252Fp%253E%250A%253Cp%2520class%253D%2522font-size-xxl%2522%253EExtra%2520Extra%2520Large%2520Text%253C%252Fp%253E"
+    class="code-snippets code-snippets-sm"
+    sandbox="allow-scripts allow-same-origin"
+  >
+  </iframe>
+
+  <h2 class="margin-vertical-3">Headings</h2>
+  <div class="typography-container margin-vertical-3">
+    <div>
+      <h6>H6 Heading</h6>
+      <h5>H5 Heading</h5>
+      <h4>H4 Heading</h4>
+      <h3>H3 Heading</h3>
+      <h2>H2 Heading</h2>
+      <h1>H1 Heading</h1>
+    </div>
+    <div>
+      <p class="h6">H6 Heading</p>
+      <p class="h5">H5 Heading</p>
+      <p class="h4">H4 Heading</p>
```

Comment:
You defined heading with heading tags as well as heading classes like h1,h2. I think you should not define heading in class name and should not use them in p tag. If you have few properties for your heading just apply them to heading tags directly like 
h1{
  font-size:4rem;
}
If you want to have classes for text your use classes like text-lg,text-sm like this .

Reply:
Commit name or description should be bit more clear, 
you are not adding components in this PR, but documentation only

Reply:
Sure, will remove that in refactoring.